source/common/upstream/upstream_impl.cc

```diff
@@ -1640,6 +1645,39 @@ void ClusterImplBase::finishInitialization() {
   }
 }
 
+bool ClusterImplBase::parseDropOverloadConfig(
```

Comment:
nit: return type should be Status, and the return an error that can be assist understanding the failure reason.